cli/cmd/addressbook.go

```diff
@@ -0,0 +1,157 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
+
+	"github.com/oasisprotocol/oasis-sdk/cli/addressbook"
+	"github.com/oasisprotocol/oasis-sdk/cli/config"
+	"github.com/oasisprotocol/oasis-sdk/cli/table"
+)
+
+var (
+	addressBookCmd = &cobra.Command{
+		Use:   "addressbook",
+		Short: "Manage addresses in the local address book",
+	}
+
+	abListCmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List addresses stored in address book",
+		Args:    cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := config.Global()
+			table := table.New()
+			table.SetHeader([]string{"Label", "Address"})
+
+			var output [][]string
+			for name, acc := range cfg.AddressBook.All {
+				output = append(output, []string{
+					name,
+					acc.Address,
+				})
+			}
+
+			// Sort output by name.
+			sort.Slice(output, func(i, j int) bool {
+				return output[i][0] < output[j][0]
+			})
+
+			table.AppendBulk(output)
+			table.Render()
+		},
+	}
+
+	abAddCmd = &cobra.Command{
+		Use:   "add <name>",
```

Comment:
Would be nice if this also supported the `add <name> <address>` variant. Actually why would you even support the survey-based one given that there is only one question and the answer is the address string? This also goes with simplifying the `Add` method (e.g. no need for import source etc.).